Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ybc"
-version = "0.1.6"
+version = "0.1.7"
```

Comment:
Let's bump this to `0.2.0` as it is a breaking set of changes both for the API as well as the dependencies.

Reply:
Absolutely sir, done in https://github.com/thedodd/ybc/pull/20/commits/ed0d26dcc0a94633d57f111a1bc80884fd1caf18